include/dlaf/communication/rdma.h

```diff
@@ -54,7 +54,7 @@ auto prepareSendTile(hpx::shared_future<matrix::Tile<const T, D>> tile) {
 /// the CPU. This helper duplicates to the GPU if the first template parameter
 /// is a GPU device. The first template parameter must be given.
 template <Device D, typename T>
-auto handleRecvTile(hpx::future<matrix::Tile<const T, CommunicationDevice<D>::value>> tile) {
+auto handleRecvTile(hpx::future<matrix::Tile<T, CommunicationDevice<D>::value>> tile) {
```

Comment:
@msimberg is it ok if we drop this `const`? can you think of any specific case where this is needed?

I'm asking because with the `recvBcast`, the tile has to be an RW one, because it has to be passed from the user then it gets filled with data from comm, differently from `recvBcastAlloc` where the tile is created inside, filled and then returned wrapped in a RO tile.

Reply:
Definitely ok with me. It'll match both const and non-const elements after this anyway. The const was just the tightest requirement for the cases where I put `handleRecvTile` into use, but it's not really a requirement.

Reply:
I think I'm going to revert it for the moment, together with changing `recvBcast` to return a `Tile<const T>` instead of a `Tile<T>`.

I was thinking about a case ahead of time, so it would be better to leave it as it is now for the moment, and just in case a specific requirement will come up, we may evaluate the change. Thanks for the feedback!